lib/maths/CBoostedTreeImpl.cc

```diff
@@ -231,7 +231,7 @@ void CBoostedTreeImpl::train(core::CDataFrame& frame,
 
     m_TrainingProgress.progressCallback(m_Instrumentation->progressCallback());
 
-    std::int64_t lastMemoryUsage(this->memoryUsage());
+    std::int64_t lastMemoryUsage{static_cast<std::int64_t>(this->memoryUsage())};
```

Comment:
The changes in this file seem unrelated. Are they intentional? (just checking they didn't pop here from another piece of work).

Reply:
I noticed some compile warnings from these statements when I added some debug. While I could factor this out into a separate change, since it is all on a branch which we'll squash when we merge I'm not sure it is worthwhile.